test/fixtures/posts.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  url: MyString
```

Comment:
What are these fixtures doing?


The error from the post systems test says `#<ActionController::RoutingError: No route matches [GET] "/posts/MyString">`

So I searched for `MyString` within the code base and found it here. 

If we comment out these fixtures, the test fails with a different error message.